common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -160,7 +160,7 @@ public class CommonValue {
                         + "\" : {\"type\": \"flat_object\"},\n"
                         + "      \""
                         + AbstractConnector.ACTIONS_FIELD
-                        + "\" : {\"type\": \"flat_object\"}\n";
+                        + "\" : {\"type\": \"flat_object\"},\n";
```

Comment:
Adding comma here makes the `ML_CONNECTOR_INDEX_FIELDS` isn't a json, did we test this change?

Reply:
Fixed.